pkgs/development/python-modules/nodriver/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  deprecated,
+  mss,
+  websockets,
+  setuptools,
+}:
+
+buildPythonPackage {
+  pname = "nodriver";
+  version = "0.32";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ultrafunkamsterdam";
+    repo = "nodriver";
+    rev = "cbb8e9ac26b70310080464d2ad43a1e2de2bfbaa";
+    hash = "sha256-FPZ+SQ7ToJMC8Wg3cAhrnuA+rqtpnjj/yLaFHIYWg8w=";
+  };
+
+  disabled = pythonOlder "3.9";
+
+  dependencies = [
+    deprecated
+    mss
+    websockets
+  ];
+
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "nodriver" ];
+  # no tests in upstream
+  doCheck = false;
+
+  meta = {
+    homepage = "https://github.com/ultrafunkamsterdam/nodriver";
+    changelog = "https://github.com/ultrafunkamsterdam/nodriver";
```

Comment:
```suggestion
```
It seems that the upstream has no changelog.

Besides this, lgtm

Reply:
Yeah seems like it. Not sure if this is relevant, but there seems to be a somewhat hidden documentation page: https://ultrafunkamsterdam.github.io/nodriver/index.html

Not sure if there is a meta key for "Documentation" or anything.

Reply:
removed the changelog, the documentation is accessible from github homepage so should be fine, i didnt see a meta attribute for "Documentation" unfortunately